website/events/templates/events/admin/registrations_table.html

```diff
@@ -39,23 +41,24 @@
                 <td>{{ registration.name }}</td>
                 <td></td>
                 {% endif %}
+                {% if event.payment_required %}
                 <td>
                     <select name="payment"
                             data-none="no_payment"
                             {% if registration.payment %}class="paid"{% endif %}>
                         <option value="no_payment"
                                 {% if not registration.payment %}selected{% endif %}>{% trans 'None' %}</option>
                         <option value="{{ payment.CARD }}"
-                                {% if registration.payment and registration.payment.type == payment.CARD %}selected{% endif %}>{% trans 'Card' %}</option>
+                                {% if registration.payment and registration.payment.type == payment.CARD %}selected{% endif %}>{% trans 'Card' %} (€ {% if registration.payment %}{{ registration.payment.amount }}{% else %}{{ event.price }}{% endif %})</option>
```

Comment:
Why?

Reply:
I always felt like it is weird that you don't see the amount when selecting that something is paid. I think it is a nice confirmation.

Reply:
But the price is the same for everyone? The event organiser knows exactly what the price should be.

Moreover, this adds more text to the event admin page that is not really necessary. I haven't checked mobile but it may be forcing other things out of view.

Reply:
Hmm you're right about the problem on mobile